src/main/java/com/org/trustservice/dto/DepartmentTrustGroupsDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.org.trustservice.dto;
+
+import java.util.List;
+import java.util.UUID;
+
+import lombok.Data;
+
+@Data
```

Comment:
Data has other implementations that are not generally required.
Stick to Setter and Getter wherever possible